src/core/lib/address_utils/parse_address.cc

```diff
@@ -20,6 +20,9 @@
 
 #include "src/core/lib/address_utils/parse_address.h"
 
+#ifdef GRPC_HAVE_VSOCK
```

Comment:
You need to include `src/core/lib/iomgr/port.h` before this line for `GRPC_HAVE_VSOCK` to be defined.

Reply:
yes, I will move the `#include "src/core/lib/iomgr/port.h"` to `parse_address.h`

Reply:
uhmm, the auto-fix(https://github.com/YadongQi/grpc/pull/18/files) suggest to move it back... how to handle this?

Reply:
Commented below.